lib/rory/request_parameter_logger.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../rory/parameter_filter'
```

Comment:
Again, no need to back out of the directory, just `require_relative 'parameter_filter'`